pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -498,6 +515,15 @@ func (l *listUsersQuery) expandTTU(
 			return err
 		}
 
+		condEvalResult, err := eval.EvaluateTupleCondition(ctx, tupleKey, typesys, req.GetContext())
+		if err != nil {
+			errs = multierror.Append(errs, err)
+			continue
+		}
+		if !condEvalResult.ConditionMet {
+			continue
+		}
```

Comment:
can we add unit tests that enter both of these conditions please?

Reply:
added in https://github.com/openfga/openfga/pull/1515/commits/e9d876627f1c37c795ce0aa3ae97d82f939eb73f